src/components/ModalBase.vue

```diff
@@ -47,8 +52,40 @@ export default defineComponent({
             type: Boolean,
             required: true,
         },
+        autoHideOnEscape: {
+            type: Boolean,
+            default: true,
+        },
+        showBackdrop: {
+            type: Boolean,
+            default: true,
```

Comment:
`showBackdrop` isn't really used, for now, this was just an experiment with modals without a backdrop.